### PR TITLE
 xGov-208 - DigiLeaf Electric Vehicle Chargers

### PR DESCRIPTION
**Pull request by bigbulb**

### CODE REVIEW

File: Proposals/xgov-208.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+id: 208
+period: 3
+title: DigiLeaf Electric Vehicle Chargers
+author: Trenton Austin (@bigbulb)
+email: TODO
```
Comment (SudoWeezy) [inline Proposals/xgov-208.md:6]: Please add an Email address to start the contract process

File: Proposals/xgov-208.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Rewards - To owners of DigiLeaf solar hardware
+
+Charger Uptime - Connectivity and availability
+
+## Present Proposal
+
+DigiLeaf proposes to operate a (4) stall 250 Amp super charger station in Southern California
+
+### Deliverable
+TODO
```
Comment (SudoWeezy) [inline Proposals/xgov-208.md:64]: Please complete this part.

**Review by SudoWeezy — CHANGES_REQUESTED** [review]

File: Proposals/xgov-208.md (hunk 1 above)
Comment (bigbulb) [reply Proposals/xgov-208.md]: Updated

File: Proposals/xgov-208.md (hunk 2 above)
Comment (bigbulb) [reply Proposals/xgov-208.md]: Updated

**Review by SudoWeezy — APPROVED** [review]
All good thanks!

**Pull request merged**